resources/eureka/package.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Why we want to save all these state information to the repository?

Reply:
Removed.